### ISSUE TITLE
question on torch.clamp() function blocking gradient propagation

### ISSUE THREAD

**Issue by yingc1309** [MCG-NJU/EMA-VFI#27]
we tried to train the EMA model on DAVIS dataset, and found the grad of Unet(RefineNet) is None.

        tmp = self.unet(img0, img1, warped_img0, warped_img1, mask, flow, c0, c1)
        res = tmp[:, :3] * 2 - 1
        pred = torch.clamp(merged[-1] + res, 0, 1)
        import pdb
        pdb.set_trace()

Input: print(self.unet.down3.conv2[0].weight.grad)
Output: None
